### ISSUE TITLE
How to add favicon to the web page ?

### ISSUE THREAD

**Issue by Redwan74** [FormidableLabs/spectacle-boilerplate#48]
Hey there, i'm new with react js and starting with this example. So what i want i s to add a simple icon beside the web page title and i couldn't change it !. does anyone know how to change it ??
![image](https://user-images.githubusercontent.com/30727109/30005326-49c14f1e-90e0-11e7-9d42-2ba4f46b0dfe.png)

**Comment by albinekb** [comment]
Yes, one way is to find the `index.html` in your project root, look for the <head /> tags
https://github.com/FormidableLabs/spectacle-boilerplate/blob/master/index.html#L3-L8

Then follow a guide like http://www.dwuser.com/education/content/quick-tip-adding-a-favicon-to-your-website/

To add your favicon :)
